server.js

```diff
@@ -1,39 +1,110 @@
 import express from 'express'
-import bodyParser from 'body-parser'
 import cors from 'cors'
 import mongoose from 'mongoose'
+import dotenv from 'dotenv'
+import listEndpoints from 'express-list-endpoints'
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
+import booksData from './data/books.json'
 
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
+dotenv.config()
+
+const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo-books"
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
 mongoose.Promise = Promise
 
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
-const port = process.env.PORT || 8080
+//Used schema just to try it out
+const bookSchema = new mongoose.Schema({
+  bookID: Number,
+  title: {
+    type: String,
+    lowercase: true,
+  },
+  authors: {
+    type: String,
+    lowercase: true
+  },
+  average_rating: Number,
+  isbn13: Number,
+  language_code: String,
+  num_pages: Number,
+  ratings_count: Number,
+  text_reviews_count: Number
+})
+
+//Mongoose model
+const Book = mongoose.model('Book', bookSchema)
+
+// Seeds data
+if (process.env.RESET_DB) {
+  const seedDatabase = async () => {
+    await Book.deleteMany()
+
+    booksData.forEach(item => {
+      const newBook = new Book(item)
+      newBook.save()
+    })
+  }
+  seedDatabase()
+}
+
+// Defines the port the app will run on. 
+const port = process.env.PORT || 8082
 const app = express()
 
 // Add middlewares to enable cors and json body parsing
 app.use(cors())
-app.use(bodyParser.json())
+app.use(express.json())
 
-// Start defining your routes here
+// List all endpoints
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send(listEndpoints(app))
 })
 
-// Start the server
-app.listen(port, () => {
-  // eslint-disable-next-line
-  console.log(`Server running on http://localhost:${port}`)
+//Endpoint to show all books
+app.get('/books', async (req, res) => {
+  const books = await Book.find(req.query)
+  res.json(books)
 })
+
+//Endpoint to get a uniqe book id from MongoDB Compass 
+app.get('/books/:bookId', async (req, res) => {
+  try {
+    const { bookId } = req.params
+    const singleBook = await Book.findById(bookId)
```

Comment:
This is a bit confusing because of the naming! I see you have picked bookId, which is very similar to the key bookID from the data you added to the database. Therefor I did not immediately get that it was the key _id created by MongoDB that should be used in this endpoint.